uber/config.py

```diff
@@ -73,9 +73,19 @@ def make_enum(self, enum_name, section, prices=False):
                 lookup[val] = desc
 
         enum_name = enum_name.upper()
-        setattr(self, enum_name + '_OPTS', opts)
-        setattr(self, enum_name + '_VARS', varnames)
-        setattr(self, enum_name + ('' if enum_name.endswith('S') else 'S'), lookup)
+        # When loading plugin's configs, we want to make sure we don't override existing enums
+        if hasattr(self, enum_name + '_OPTS'):
```

Comment:
I think in this case, we may want to break away from Sideboard so we can make certain assumptions in regards to load config. We always want to merge all config files before parsing them and we always want to load the uber plugin first, and building these assumptions into RAMS means that 1) the ABI is more stable/you can do fewer horrible things with it and 2) making this happen becomes a heck of a lot easier.